php/composer.json

```diff
@@ -1,7 +1,31 @@
 {
-    "name": "dzcode.io/kuliya",
+    "name": "dzcodeio/kuliya",
     "description": "Algeria's college hierarchy dataset as a package.",
+    "keywords": [
+        "dzcode", "algeria", "dataset", "package", "college"
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "dzcode.io community",
+            "homepage": "https://www.dzcode.io/",
+            "role": "Developer"
+        }
+    ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^7"
+    },
+    "autoload": {
+        "psr-4": {
+            "Dzcodeio\\Kuliya\\": "src/"
```

Comment:
Most popular php projects/frameworks use `PascalCase` for file/dir names so i dont think that would cause any issues but i can use `snake_case` starting from the next PR. As for namespaces, Class/Method names and class file names i chose to respect PHP standard recommendations, specificaly [PSR-4](https://www.php-fig.org/psr/psr-4/) for class autoloading and [PSR-12](https://www.php-fig.org/psr/psr-12/) for code style which suggests adopting `PascalCase` for namespaces/classes and class file names and `camelCase` for methods.

Reply:
In that case then, it's better to keep it consistent across the pho folder, so let it be PascalCased 👌🏽.

Just note that later on, changing letter cases will cause problems on windows.

Reply:
Ahh i see! thanks for the hint, i'll investigate on it then and use the best strategy possible.